src/System Application/App/Edit in Excel/src/EditinExcelImpl.Codeunit.al

```diff
@@ -43,6 +43,7 @@ codeunit 1482 "Edit in Excel Impl."
         CreateEndpointForObjectTxt: Label 'Creating endpoint for %1 %2.', Locked = true;
         EditInExcelHandledTxt: Label 'Edit in excel has been handled.', Locked = true;
         EditInExcelOnlySupportPageWebServicesTxt: Label 'Edit in Excel only support web services created from pages.', Locked = true;
+        EditInExcelInvalidFilterErr: Label 'We had to remove the filters applied to the following fields, because they are not available in the Office Add-In. Please notice that as a result of this that the number of rows you exported to Excel could vary.\n %1', Locked = true;
```

Comment:
Since we use this in a `Message()`, should it be translated and hence not locked?

(Also, smal typo "notice that ... that").

Reply:
Oh, one more thing: the new line in the AL `Message()` function is not `\n` but rather `\`

See:
https://learn.microsoft.com/en-us/dynamics365/business-central/dev-itpro/developer/methods-auto/dialog/dialog-message-method#:~:text=Use%20a%20backslash%20(%5C)%20to%20start%20a%20new%20line

Reply:
Missing comment for %1 in newly added Label EditInExcelInvalidFilterErr